### DISCUSSION TITLE
[@types/compression-webpack-plugin]: webpack 5 compatibility

### DISCUSSION THREAD

**Discussion by nickensoul** [DefinitelyTyped/DefinitelyTyped#54412] [Issues with a @types package]
Hi.
Since webpack 5 has been released, and even while [webpack-contrib community plans to implement native TypeScript support](https://github.com/webpack-contrib/compression-webpack-plugin/issues/254), current typings in package `@types/compression-webpack-plugin` are incompatible with webpack version 5.
So it would be nice to fix type errors.

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
so what's are those errors, to be specific? 
Note that DT provides support for both Webpack v4 and v5, that can hamper TS experience
